process/smartContract/process.go

```diff
@@ -151,7 +152,7 @@ func NewSmartContractProcessor(args ArgsNewSmartContractProcessor) (*scProcessor
 	}
 
 	apiCosts := args.GasSchedule.LatestGasSchedule()[core.ElrondAPICost]
-
+	builtInFuncCost := args.GasSchedule.LatestGasSchedule()[core.BuiltInCost]
```

Comment:
if args.GasSchedule.LatestGasSchedule() or builtInFuncCost could be nil they should be checked before to access
[core.BuiltInCost] or [core.BuiltInFunctionESDTTransfer] below

Reply:
this is on constructor - if it panics it is good :)

Reply:
We might return an error instead of letting the node panic.
Maybe create a function that will test both `core.BuiltInCost` and `core.BuiltInFunctionESDTTransfer` values

Reply:
These unit tests do not make sense :) the node will not start reach this space. As it will fail in bultinfunctions factory.